docs/debug-launch.md

```diff
@@ -113,14 +113,78 @@ Here are minimal examples of a `launch.json` file that uses `cmake.launchTargetP
 }
 
 ```
+### ctest
+```jsonc
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "(ctest) Launch",
+            "type": "cppvsdbg",
+            "request": "launch",
+            // Resolved by CMake Tools:
+            "program": "${command:cmake.launchTargetPath}",
+            "args": [],
+            "stopAtEntry": false,
+            "cwd": "${workspaceFolder}",
+            "environment": [
+                {
+                    // add the directory where our target was built to the PATHs
+                    // it gets resolved by CMake Tools:
+                    "name": "PATH",
+                    "value": "${env:PATH}:${command:cmake.getLaunchTargetDirectory}"
+                },
+                {
+                    "name": "OTHER_VALUE",
+                    "value": "Something something"
+                }
+            ],
+            "console": "externalTerminal"
+        }
+    ]
+}
 
-
+```
 
 The value of the `program` attribute is expanded by CMake Tools to be the absolute path of the program to run.
 
 > **Note:**
 > You must successfully [configure](configure.md) before `cmake.launchTargetPath` and `cmake.getLaunchTargetDirectory` will resolve correctly.
 
+## Debugging tests
+
+You can also construct launch.json configurations that allow you to debug tests in the Test Explorer.
+The easiest way to do this is to construct the debug configuration using `cmake.testProgram` for the `program` field, and `cmake.testArgs` for 
+the `args` field.
+
+A couple of examples:
+
+### gdb
```

Comment:
Have you tested if this work? Do you need to specify some other fields like `MIMode`?

Reply:
It didn't make me when I tested, I didn't have any issues with this configuration, that I could see.